rpi-eeprom-update

```diff
@@ -539,7 +539,7 @@ lookupVersionInfo()
    if [ "${HAVE_VL805_EEPROM}" = 1 ]; then
       getVL805UpdateVersion
       if [ -n "${VL805_CURRENT_VERSION}" ] && [ -n "${VL805_UPDATE_VERSION}" ]; then
-         if [ "${VL805_CURRENT_VERSION}" \< "${VL805_UPDATE_VERSION}" ];  then
+         if [ "${VL805_CURRENT_VERSION}" -lt "${VL805_UPDATE_VERSION}" ];  then
```

Comment:
`-lt` compares integers, and will fail for a non-numeric version such as that used by VL805 which is correctly using `\<`.

Reply:
Maybe the VL805 version string could be converted to an integer via $((0x.